core/templates/domain/story/story-update.service.ts

```diff
@@ -65,18 +70,24 @@ export class StoryUpdateService {
       apply: StoryUpdateApply, reverse: StoryUpdateReverse): void {
     let changeDict = cloneDeep(params) as BackendChangeObject;
     changeDict.cmd = command;
-    let changeObj = new Change(changeDict, apply, reverse);
+    let changeObj = new Change(
+      changeDict, apply as changeBackendDict, reverse as changeBackendDict);
     try {
       this._undoRedoService.applyChange(changeObj, story);
-    } catch (err) {
-      this._alertsService.addWarning(err.message);
+      // The catch parameter type can only be any or unknown. The type 'unknown'
+      // is safer than type 'any' because it reminds us that we need to perform
+      // some sorts of type-checks before operating on our values.
```

Comment:
Great explanation! Just move it to left a bit.
```suggestion
    // The catch parameter type can only be any or unknown. The type 'unknown'
    // is safer than type 'any' because it reminds us that we need to perform
    // some sorts of type-checks before operating on our values.
```

Reply:
Thanks, done.